mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+// makeGubernatorLink returns a URL to view the build results in a GCS path.
+func makeGubernatorLink(id string) string {
+	return "https://k8s-gubernator.appspot.com/build" + id
+}
```

Comment:
Hm. Maybe they broke after I put them in.

Can we make gubernator load faster?

On Wed, Jun 1, 2016 at 4:10 PM, Ryan Hitchman <notifications@github.com>
wrote:

> In mungegithub/mungers/flake-manager.go
> <https://github.com/kubernetes/contrib/pull/1112#discussion_r65459496>:
>
> > @@ -127,6 +127,11 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
> >  	return true
> >  }
> >
> > +// makeGubernatorLink returns a URL to view the build results in a GCS path.
> > +func makeGubernatorLink(id string) string {
> > +	return "https://k8s-gubernator.appspot.com/build" + id
> > +}
>
> All the links are broken.
>
> Right now Gubernator doesn't have very much additional information versus
> what this puts in the issue, but it's intended to provide a better triaging
> experience than poking around in the GCS buckets directly.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1112/files/045e17ab2b998712c9d04de10fb3180206085e99#r65459496>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglmUuSJcQRJWSzKXyxr4EAESkJOtZks5qHhFWgaJpZM4IrErK>
> .
>